src/bp/ui-studio/src/web/palette.scss

```diff
@@ -84,6 +84,27 @@ $fontFamily: Roboto;
   --c-disabled: #eee;
   --c-positive: #80c18d;
   --c-negative: #d1495a;
+
+  // Albert variables
+  // We should start using these variables from now on and phase out the rest of this file
+  --white: #ffffff;
+  --seashell: #f5f6f8;
+  --gray: #bec5c9;
+  --reef: #5c7080;
+  --shark: #1a1e22;
+  --overlay: rgba(26, 30, 34, 0.8);
+  --say: #a4ded0;
+  --lighthouse: #f66f48;
+  --hover-lighthouse: #f39c82;
+  --hover-lighthouse-30: #f9ddd5;
+  --ocean: #3276ea;
+  --hover-ocean: #cad7f5;
+
+  --spacing-x-small: 2px;
+  --spacing-small: 4px;
+  --spacing-medium: 8px;
+  --spacing-large: 12px;
+  --spacing-x-large: 16px;
```

Comment:
I just saw that we have the palette twice in different locations, could we have it once in ui-shared, then import the scss here and in the admin ? It would be easy to forget to update one place or the other.

Reply:
I couldn't figure out how to import the file from a library easily, I suggest to leave it like this for now since we will need this feature in order to complete what's needed for 12.9 and try to fix it in the near future. We already have variables that are shared in both locations atm